src/Microsoft.AspNet.Mvc.Razor/RazorTextWriter.cs

```diff
@@ -186,24 +189,24 @@ public override void Flush()
             {
                 IsBuffering = false;
                 TargetWriter = UnbufferedWriter;
-                await CopyToAsync(UnbufferedWriter);
+                await CopyToAsync(UnbufferedWriter, HtmlEncoder);
             }
 
             await UnbufferedWriter.FlushAsync();
         }
 
         /// <inheritdoc />
-        public void CopyTo(TextWriter writer)
+        public void CopyTo(TextWriter writer, IHtmlEncoder encoder)
         {
             writer = UnWrapRazorTextWriter(writer);
-            BufferedWriter.CopyTo(writer);
+            BufferedWriter.CopyTo(writer, encoder);
         }
 
         /// <inheritdoc />
-        public Task CopyToAsync(TextWriter writer)
+        public Task CopyToAsync(TextWriter writer, IHtmlEncoder encoder)
         {
             writer = UnWrapRazorTextWriter(writer);
-            return BufferedWriter.CopyToAsync(writer);
+            return BufferedWriter.CopyToAsync(writer, encoder);
         }
```

Comment:
This is the place it is used - https://github.com/aspnet/Mvc/pull/2797/files#diff-21281ad55491937a33b7eee9b50bdc81L278